components/pedigree/ui/src/main/resources/PhenoTips/PedigreeMacros.xml

```diff
@@ -50,7 +50,14 @@
   #set ($cancerColors = $pedigreeDataJSON.settings.colors.cancers)
 
   #if (($disorderColors &amp;&amp; $disorderColors.size() &gt; 0) || ($geneColors &amp;&amp; $geneColors.size() &gt; 0) || ($cancerColors &amp;&amp; $cancerColors.size() &gt; 0))
-    #set ($listyle="margin: 0 .5em 0 0; font-family: FreeSerif, serif;")
+    #if ($xcontext.action == 'export')
+      #set ($circle = "⬤")
+      #set ($listyle = "margin: 0 .5em 0 0; font-family: FreeSerif, serif;")
+    #else
+      #set ($circle = "&lt;span class=""fa fa-circle"">&lt;/span>")
```

Comment:
Yes. Below is Chrome41/OSX10.10:
![p0000001___phenotype_database](https://cloud.githubusercontent.com/assets/4006062/7145401/0ab727d4-e2ba-11e4-844b-a509f04642f6.png)